### PR TITLE
Feature: Related products grid

### PR DESCRIPTION
**Pull request by acdavitt**
-Added grid structure
-Rendered some dummy data on page for RelatedProducts
-Did not invest too much time into formatting with Grid because I want to see how Carousel will fit in first
-Started with dummy data just for Related Products and left placeholder in for Outfit List

### CODE REVIEW

**Review by ChrisRPeterson — APPROVED** [review]
Looks good!

File: client/components/RelatedProducts/Related/RelatedProductsList.jsx [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 
-const RelatedProductsList = () => {
-  //get request for related product ids based on id of current product
-  //then get request for each product to product_id (for name, category, default_price) AND get request to id/styles (for sale_price and image) as well??
-  return <RelatedProductCard />;
+
+const RelatedProductsList = (props) => {
+
+  return  (
+    <GridList cols={4}>
```
Comment (julia-thea) [inline client/components/RelatedProducts/Related/RelatedProductsList.jsx:11]: Nice usage of grid list. Neat mapping of props items to RelatedProductCard 👍

File: client/components/RelatedProducts/Related/RelatedProductsList.jsx [hunk 2]
```diff
@@ -1,10 +1,17 @@
-import React from 'react';
+import React, {useState} from 'react';
```
Comment (julia-thea) [inline client/components/RelatedProducts/Related/RelatedProductsList.jsx:1]: I see importing useState here but not using it, I assume it's for later usage ☺️

**Review by julia-thea — COMMENTED** [review]
Good starting point for the components layout. Definitely appreciate adding some dummy data to help visualize what the actual widget will look like

**Pull request merged**